tools/lint-pr-url.mjs

```diff
@@ -28,7 +28,7 @@ for await (const line of diff) {
   } else if (!validatePrUrl(line.match(prUrlDefinition)?.[1])) {
     console.warn(
       `::warning file=${currentFile},line=${currentLine++},col=${line.length}` +
-      '::pr-url doesn\'t match the actual PR URL.'
+      '::pr-url doesn\'t match the current PR actual URL.'
```

Comment:
```suggestion
      '::pr-url doesn\'t match the current PR URL.'
```
Is `actual` needed here?

Reply:
I'm not sure, I'm not a native speaker 😅  I wanted to make sure "current" was describing "PR" and not "URL". Can I say:
```suggestion
      '::pr-url doesn\'t match the current PR's URL.'
```

Reply:
:)
I don't think `'s` is used for non-living things. Maybe we can say this:
```suggestion
      '::pr-url doesn\'t match the URL of the current PR.'
```